docs/Nachet_inference_documentation.md

```diff
@@ -0,0 +1,112 @@
+# Inference Request with n pipelines
+
+## Glossary
+
+### Pipelines
+Pipelines are define as a set of models that follow each others where output of
+one model is used as input for the next models. A pipeline contains from 1 to n
+models.
+
+```mermaid
+flowchart LR
+
+A{Pipeline entry point}-->|user input|SB1-->F{Pipeline exit point}-->|"last model
+output"|G>"return
+result.json to user"]
+
+subgraph SB1[In pipeline]
+    direction TB
+
+    B(model 01)-->|"model output
+    send to the next one"|C(model 02)-->|can have n models|D(n models)
+
+end
+```
+
+### Models
+A model is a AI model who's part of a pipeline. A model accept images as input
+and return json as output. Generally json contains coordinate of object in the
+source image, that he may prompt out to feed the next step of the model.
+
+
+### Model from Frontend
```

Comment:
On the frontend interface, a pipeline will be called a model, because the user will
not be aware of the difference. From the user's perspective, they sent data to a
model and received the result.

Reply:
Done